src/styles/_catalog.scss

```diff
@@ -1,22 +1,38 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $width-card);
+  justify-content: center;
+  gap: $gap-card;
+  margin: 0, auto;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    & {
```

Comment:
you don't need `&` here

Reply:
it's a scss feature to write media queries inside nesting
`.catalog {
  @media () {
    properties
  }
}`